src/movies.js

```diff
@@ -1,16 +1,51 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
+
+const movies = require('./data');
+
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+function getAllDirectors(movies) {
+  const allDirectors = movies.map(function (film) {
+    return film.director;
+  });
```

Comment:
If you clearly understand this syntax, start practising arrow functions, shorter syntax and you'll use it more in the future ;)
```
 const allDirectors = movies.map((film) => film.director);
```

If it is only one line of code after arrow, you do not need curly brace and return.